docker-jans-auth-server/scripts/upgrade.py

```
@@ -80,6 +80,43 @@ def _transform_lock_dynamic_config(conf, manager):
+        "CN_LOCK_GRPC_SERVER_MODE" in os.environ,
+        "CN_LOCK_GRPC_PORT" in os.environ,
+    ]):
+        logger.warning(
+            "Detected experimental env vars CN_LOCK_GRPC_SERVER_MODE or CN_LOCK_GRPC_PORT. "
+            "It is recommended to use the default gRPC server mode and port by removing these env vars."
+        )
+
+    try:
+        grpc_port = int(grpc_port)
+    except (TypeError, ValueError):
+        logger.warning(f"Found invalid value for CN_LOCK_GRPC_PORT={grpc_port}; fallback to default port {jetty_port}")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)